### PR TITLE
Merge from master

### PR DESCRIPTION
**Pull request by ValMenn**

### CODE REVIEW

**Comment by rainersigwald** [comment]
Any interesting merge conflicts we should look at more closely?

**Comment by rainersigwald** [comment]
If not, go ahead :)

File: src/dir.targets [hunk 1]
```diff
@@ -1,12 +1,21 @@
 ﻿<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\dir.targets" />
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" Condition="'$(NetCoreBuild)' != 'true'"/>
-
-  <Import Project="$(MSBuildExtensionsPath32)\Microsoft\Portable\$(TargetFrameworkVersion)\Microsoft.Portable.CSharp.targets" Condition="'$(NetCoreBuild)' == 'true'"/>
-  <PropertyGroup Condition="'$(NetCoreBuild)' == 'true'">
-    <NuGetTargetMoniker>DNXCore,Version=v5.0</NuGetTargetMoniker>
-  </PropertyGroup>
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
```
Comment (AndyGerlicher) [inline src/dir.targets]: This looks wrong. I added the AfterBuild and sign.targets, but the others are for .NET core stuff Daniel added. Looks like the merge removed those.

**Comment by AndyGerlicher** [comment]
LTGM except for the dir.targets resolved incorrectly I believe. Changes I made look good though.

**Comment by ValMenn** [comment]
There was nothing special, just tedious. I address Andy's comments and fixed src\dir.targets.

**Pull request merged**